common.gypi

```diff
@@ -389,7 +389,6 @@
           'GCC_DYNAMIC_NO_PIC': 'NO',               # No -mdynamic-no-pic
                                                     # (Equivalent to -fPIC)
           'GCC_ENABLE_CPP_EXCEPTIONS': 'NO',        # -fno-exceptions
-          'GCC_ENABLE_CPP_RTTI': 'NO',              # -fno-rtti
```

Comment:
Could you try if it works by keeping this one and then "correcting" it later on for chakracore? Less changes if that works.